specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingRequest.json

```diff
@@ -0,0 +1,504 @@
+          "description": "The ID that uniquely identifies a billing profile.",
+          "type": "string",
+          "readOnly": true
+        },
+        "billingProfileDisplayName": {
+          "description": "The name of the billing profile.",
+          "type": "string",
+          "readOnly": true
+        },
+        "createdBy": {
+          "description": "UPN of user who created the request.",
+          "type": "string",
+          "readOnly": true
+        },
+        "creationDate": {
+          "description": "The date and time when the request was created.",
+          "type": "string",
+          "readOnly": true
+        },
+        "expirationDate": {
+          "description": "The date and time when the request expires.",
+          "type": "string",
+          "readOnly": true
+        },
+        "decisionReason": {
+          "description": "The reason to approve or decline the request.",
+          "type": "string"
+        },
+        "invoiceSectionId": {
+          "description": "The fully qualified ID that uniquely identifies an invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "invoiceSectionName": {
+          "description": "The ID that uniquely identifies an invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "invoiceSectionDisplayName": {
+          "description": "The name of the invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "customerId": {
+          "description": "The fully qualified ID that uniquely identifies a customer.",
+          "type": "string",
+          "readOnly": true
+        },
+        "customerName": {
+          "description": "The ID that uniquely identifies a customer.",
+          "type": "string",
+          "readOnly": true
+        },
+        "customerDisplayName": {
+          "description": "The name of the customer.",
+          "type": "string",
+          "readOnly": true
+        },
+        "subscriptionId": {
+          "description": "The fully qualified ID that uniquely identifies a billing subscription.",
+          "type": "string",
+          "readOnly": true
+        },
+        "subscriptionName": {
+          "description": "The ID that uniquely identifies a billing subscription.",
+          "type": "string",
+          "readOnly": true
+        },
+        "subscriptionDisplayName": {
+          "description": "The name of the billing subscription.",
+          "type": "string",
+          "readOnly": true
+        },
+        "justification": {
+          "description": "Justification for submitting request.",
+          "type": "string"
+        },
+        "recipients": {
+          "description": "The recipients of the billing request.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "x-ms-identifiers": []
+        },
+        "scope": {
```

Comment:
requestScope (write property)
billingScope (read-only property that may be derived for the service)